src/openzeppelin/tests/security/test_pausable.cairo

```diff
@@ -1,47 +1,107 @@
-use openzeppelin::tests::mocks::mock_pausable::MockPausable;
+use openzeppelin::security::pausable::Pausable;
+use openzeppelin::security::pausable::Pausable::PausableImpl;
+use openzeppelin::security::pausable::Pausable::InternalImpl;
+
+fn STATE() -> Pausable::ContractState {
+    Pausable::contract_state_for_testing()
+}
+
+//
+// is_paused
+//
 
 #[test]
 #[available_gas(2000000)]
-fn test_pause_when_unpaused() {
-    assert(!MockPausable::is_paused(), 'Should not be paused');
-    assert(MockPausable::get_count() == 0, 'Should be 0');
-    MockPausable::assert_unpaused_and_increment();
-    assert(MockPausable::get_count() == 1, 'Should increment');
-    MockPausable::pause();
-    assert(MockPausable::is_paused(), 'Should be paused');
+fn test_is_paused() {
+    let mut state = STATE();
```

Comment:
```suggestion
    let state = STATE();
```

Reply:
We need this to be mutable to call `_pause` and `_unpause` in this test